distmv/statdist.go

```diff
@@ -12,6 +12,14 @@ import (
 	"github.com/gonum/stat"
 )
 
+type DistNormaler interface {
```

Comment:
(playing devil's advocate to understand policy)

With `mat64` (in my perception), it was not clear that the interfaces were useful and sufficient. There's a similar problem with `distxv`, where there are tons of possible interfaces, and it's not fully clear which of them are powerful enough to merit inclusion. Here seems different. The types are clearly designed to a certain interface, and any code that wants to abstract over a distance metric will want to use this interface. Having the interface in `distmv`, rather than my package, allows different codes to communicate through the same interface instead of needing wrapper types (https://github.com/golang/go/issues/8082)